### PR TITLE
Add d8 upgrade info to Major Version Upgrade doc

### LINKED ISSUE #929: Major Version Drupal Upgrades Doc Update
Re: [Major Version Drupal Upgrades](https://pantheon.io/docs/articles/drupal/major-version-drupal-upgrades)
## Issue Description:
D8 puts migrate.module in core and no longer supports upgrading through update.php
## Suggested Resolution 
Add D8 upgrade instructions

See this article for reference: 
https://www.drupal.org/node/2350521

### PR DESCRIPTION
**Pull request by nataliejeremy**
Closes #929 
@ttrowell @davidstrauss Please review--thanks.

### CODE REVIEW

File: source/docs/articles/drupal/major-version-drupal-upgrades.md [hunk 1]
```diff
@@ -6,26 +6,42 @@ category:
...
+
+## Upgrade to Drupal 8
+
+In Drupal 8, the migrate module is now in core and no longer supports upgrading through update.php.
+
+1. Start a new site using Drupal 8 as the start state.
+2. Add the 8.x version of your contrib modules.
+3. Import your existing data.
+4. Debug, QA, release.
+
```
Comment (rachelwhitton) [inline source/docs/articles/drupal/major-version-drupal-upgrades.md]: Suggest to expand instructions, either within this doc or link to the proposed doc in #923 when it has been created. 

> Import your existing data. 

We should note how to create the imports, and the 2 options for importing. May be worth generating some screenshots for the new Migrate API. See [Upgrading from Drupal 6 or 7 to Drupal 8](https://www.drupal.org/upgrade/migrate).

File: source/docs/articles/drupal/major-version-drupal-upgrades.md [hunk 2]
```diff
@@ -6,26 +6,42 @@ category:
...
-## Content and Configuration
+### Content and Configuration
 
 While you can try to get Drupal to handle all the data architecture changes between major revisions (importing the old database and running update.php), this is often not a complete solution. Depending on the specific module stack and configuration of your current site, it may be faster and more direct to plan and execute a content migration to the new site rather than trying to use the built-in update tools.
 
 If you are not having much luck with update.php, consider setting up the new site and using tools like the migrate.module to import your existing content. While this might initially seem like more work, it can often lead to a cleaner result more quickly, especially if your new site includes major architectural changes, features, or a redesign.
+
+## Upgrade to Drupal 8
+
+In Drupal 8, the migrate module is now in core and no longer supports upgrading through update.php.
```
Comment (rachelwhitton) [inline source/docs/articles/drupal/major-version-drupal-upgrades.md]: Suggest rephrasing to: 

>Drupal 8 does not support `update.php` or upgrading from a previous major versions. Instead, users must migrate content and configurations from an existing Drupal 6 or 7 site into a fresh Drupal 8 installation.

Migrations from D7 are not fully supported (yet). D6 sites should okay to migrate. The Migrate Module is still in development. See [Introduction to migrate D6 to D8](https://www.drupal.org/node/2350521) and [Upgrading from Drupal 6 or 7 to Drupal 8](https://www.drupal.org/upgrade/migrate)

Comment (rachelwhitton) [inline source/docs/articles/drupal/major-version-drupal-upgrades.md]: Actually, the above is not accurate. `update.php` is still around, just not used for upgrading to D8 from previous major versions (D6/D7 to D8). It's used for minor upgrades (8.x.x)

Suggest:
> Drupal 8 does not support upgrading from previous major versions. Instead, users must migrate content and configurations from an existing Drupal 6 or 7 site into a fresh Drupal 8 installation. `update.php` is now used exclusively for upgrading within the same version (8.x.x).

File: source/docs/articles/drupal/major-version-drupal-upgrades.md [hunk 3]
```diff
@@ -6,26 +6,42 @@ category:
   - managing
 keywords: drupal, upgrade, upgrading, revision upgrade, update, updating
 ---
-The best practice for doing a major Drupal revision upgrade (e.g. version 6 to version 7) is to start a new site. Even the simplest of upgrades require their own QA and deployment process, and trying to do an upgrade on an existing site is not a recipe for success.
+The best practice for doing a major Drupal revision upgrade (i.e. version 6 to version 7) is to start a new site. Even the simplest of upgrades require their own QA and deployment process, and trying to do an upgrade on an existing site is not a recipe for success.
```
Comment (bmackinney) [inline source/docs/articles/drupal/major-version-drupal-upgrades.md]: It's the only way, not just a best practice. `To upgrade Drupal to a new major version...`

File: source/docs/articles/drupal/major-version-drupal-upgrades.md [hunk 4]
```diff
@@ -6,26 +6,42 @@ category:
...
 
 Also, Pantheon needs to track the proper upstream Git history for your site to deliver core updates. By starting a new site for the upgrade, you ensure that future core updates will be available via the Dashboard.
 
-## Set up the Code and Migrate the Data
+The best way to think about upgrading to the next major revision of Drupal on Pantheon is the same as when you get a new laptop or smartphone: first you set up both devices side-by-side, then you migrate the data from the old to the new.
```
Comment (bmackinney) [inline source/docs/articles/drupal/major-version-drupal-upgrades.md]: Clarify, "set up both devices side-by-side," - doe it mean, "Install all of the applications on the new machine,"?

File: source/docs/articles/drupal/major-version-drupal-upgrades.md (hunk 2 above)
Comment (bmackinney) [inline source/docs/articles/drupal/major-version-drupal-upgrades.md]: The process is generally the same as from 6-7, but the migrate module needed in D7 (import your existing data) is in core in D8. update.php is used for a variety of purposes, and shouldn't need to be mentioned here.

File: source/docs/articles/drupal/major-version-drupal-upgrades.md [hunk 5]
```diff
@@ -6,26 +6,42 @@ category:
...
+1. Start a new site using Drupal 8 as the start state.
+2. Add the 8.x version of your contrib modules.
+3. Import your existing data.
+4. Debug, QA, release.
+
+### Content and Configuration
+
+Drupal 8 migrations automatically create the needed content types and establish the mappings between the old and new fields by default.
+
+If needed, you can customize your migration using the included hooks or use the configuration schema to use the included plugins with your custom data set. The hooks in Drupal 8 let you alter data in the prepareRow stage without creating a custom migration to handle the data.
```
Comment (bmackinney) [inline source/docs/articles/drupal/major-version-drupal-upgrades.md]: Add link to more info.

File: source/docs/articles/drupal/major-version-drupal-upgrades.md (hunk 4 above)
Comment (nataliejeremy) [inline source/docs/articles/drupal/major-version-drupal-upgrades.md]: Both of your comments are on existing content (I didn't edit or add that).

File: source/docs/articles/drupal/major-version-drupal-upgrades.md (hunk 1 above)
Comment (nataliejeremy) [inline source/docs/articles/drupal/major-version-drupal-upgrades.md]: Can you add that info? @rachelwhitton

**Comment by rachelwhitton** [comment]
@nataliejeremy @bmackinney please review commit https://github.com/pantheon-systems/documentation/commit/dc88e3c1e70c842f18eed99b3c6c1a6fdfdc37b3

**Comment by nataliejeremy** [comment]
+1

**Comment by bmackinney** [comment]
:+1:

**Comment by rachelwhitton** [comment]
Completed in https://github.com/pantheon-systems/documentation/commit/f106882482b3d1a8b765c0647a75505eabd7898f within #999

**Pull request closed without merging**